### PR TITLE
Change to weak comparison

### PR DESCRIPTION
**Pull request by redferret**
## What is the new behavior?

- Changes one line of code to use weak comparator instead of strong comparisons with the authorization mall 
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/actions/auth_actions/confirm_login_action.js [hunk 1]
```diff
@@ -15,6 +14,7 @@ Actions.register(CONFIRM_LOG_IN, payload => {
   })
   .then(checkStatus)
   .then(response => {
+    console.log(response);
```
Comment (mkrumholz) [inline src/actions/auth_actions/confirm_login_action.js:17]: @redferret don't forget to remove this!

File: src/components/google_sign_in.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ export default function GoogleSignIn() {
     let userIsAuthed = googleAuthStore.isAuthed();
 
     auth.setUserAuthedState(userIsAuthed, () => {
-      if (authMall.isUserNew() === 'true') {
+      if (authMall.isUserNew() == true) {
```
Comment (mkrumholz) [inline src/components/google_sign_in.js:34]: Thanks for making the update!

**Review by mkrumholz — APPROVED** [review]

File: src/actions/auth_actions/confirm_login_action.js (hunk 1 above)
Comment (twarbelow) [reply src/actions/auth_actions/confirm_login_action.js]: removed.

**Review by twarbelow — APPROVED** [review]
works fine on my end. I removed the console.log

**Pull request merged**